drizzlepac/drizCR.py

```diff
@@ -7,35 +7,34 @@
 :License: :doc:`LICENSE`
 
 """
-from __future__ import absolute_import, division, print_function # confidence medium
+from __future__ import absolute_import, division, print_function
```

Comment:
Simply curious: do we still need these imports if we are only supporting Python 3.x?

Reply:
@stsci-hack I am not sure what _"we are only supporting Python 3.x"_ really means since I see `drizzlepac` is still available in `astroconda` with `Python 2.7`. Is it not? If it still is, I suppose it better work. Feel free to investigate this issue in more depth but from my point of view, it just does not hurt to leave it there. The effort to remove these `from __future__` imports should be minimal "in future".

Reply:
If you are only supporting Python 3, the code throws an error when you try to **install** in Python 2 **and** it is no longer released in any Python 2 environments.